src/Abp/Notifications/NotificationPublisher.cs

```diff
@@ -87,12 +88,12 @@ public static int[] AllTenants
                 NotificationName = notificationName,
                 EntityTypeName = entityIdentifier == null ? null : entityIdentifier.Type.FullName,
                 EntityTypeAssemblyQualifiedName = entityIdentifier == null ? null : entityIdentifier.Type.AssemblyQualifiedName,
-                EntityId = entityIdentifier == null ? null : entityIdentifier.Id.ToJsonString(),
+                EntityId = entityIdentifier == null ? null : entityIdentifier.Id.ToJsonString(referenceLoopHandling: ReferenceLoopHandling.Ignore),
```

Comment:
How entityIdentifier can cause a reference loop exception ? It doesn't seem to be possible.